content/02.introduction.md

```diff
@@ -67,14 +72,30 @@ Coronaviruses are known to cause respiratory illnesses in humans through the fol
 *Presentation of COVID-19:* 
 Information is rapidly becoming available about the wide range of symptoms that can be associated with COVID-19 as well as the range of symptom severity, onset from exposure, and possible risk or protective factors...
 
-In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
+In a retrospective sample of COVID-19 patients in Wuhan observed during December 2019 and January 2020, [@doi:10/ggnxb3] described the clinical features of patients infected with SARS-CoV-2 including lower respiratory tract infection with fever, dry cough, and dyspnea. 
+[@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common which suggests that the virus targets cells located in the lower respiratory tract. 
+Recent reports suggest that symptoms of infection by SARS-CoV-2 can vary greatly, making it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate infection and should prompt isolation. 
+[@doi:10/ggnxb3] also found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment score, as well as high levels of d-dimer. 
 Mortality also appeared to be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
 They also found that survivors continued to shed the virus for a median of 20 days and a maximum of at least 37 days.
 
-
 #### Vaccines for Viruses: Strategies for and challenges to development
 
-What information is needed to develop a vaccine? How have vaccines for other viruses such as H1N1 been developed?
+Generally, the first step in producing a vaccine is characterizing the target.
+The genetic sequence of SARS-CoV-2 was released to the world on January 11, 2020, which launched an intense global effort to develop a vaccine to prevent COVID-19.
```

Comment:
```suggestion
The genetic sequence of SARS-CoV-2 was published on January 11, 2020, which aided the global effort to develop a vaccine to prevent COVID-19.
```

Reply:
The genetic sequence is not the reason why we try to develop a vaccine...but it helped!